### ISSUE TITLE
ob-ipython hangs with "Connecting to host localhost:9988" displayed in minibuffer

### ISSUE THREAD

**Issue by venkat0791** [gregsexton/ob-ipython#82]
I'm using ob-ipython-20160703.639 (from Melpa) in Emacs 24.5, on a Windows 7 computer. When I attempt to run an ipython source block in an org buffer (with C-c C-c), ob-ipython hangs with the message "Connecting to host localhost:9988" displayed in the minibuffer. Further, in the *ob-ipython-client-driver* buffer, I get the following error message:

```
Traceback (most recent call last):
  File "c:/Users/sivarama/.emacs.d/elpa/ob-ipython-20160703.639/driver.py", line 167, in <module>
    main(sys.argv)
  File "c:/Users/sivarama/.emacs.d/elpa/ob-ipython-20160703.639/driver.py", line 163, in main
    app.listen(args.port)
  File "c:\Users\sivarama\Applications\Continuum\Anaconda3\lib\site-packages\tornado\web.py", line 1825, in listen
    server.listen(port, address)
  File "c:\Users\sivarama\Applications\Continuum\Anaconda3\lib\site-packages\tornado\tcpserver.py", line 126, in listen
    sockets = bind_sockets(port, address=address)
  File "c:\Users\sivarama\Applications\Continuum\Anaconda3\lib\site-packages\tornado\netutil.py", line 196, in bind_sockets
    sock.bind(sockaddr)
OSError: [WinError 10048] Only one usage of each socket address (protocol/network address/port) is normally permitted

Process client-driver exited abnormally with code 1
```

Some (possibly relevant) settings in my init.el:
1. ob-ipython-command is set to "jupyter" (I'm running 4.1.1, from Anaconda);
2. python-shell-interpreter is set to "python";
3. python-shell-interpreter-args is set to "-i C:\\Users\\sivarama\\Applications\\Continuum\\Anaconda3\\Scripts\\jupyter-console-script.py".

Any idea what might be going wrong here? Any further information I should provide about my setup?

**Comment by gregsexton** [comment]
The error is in that traceback. It can't bind to the port. Is the driver already running? Do you have something already on that port? Try killing whatever is there and retrying.

**Comment by venkat0791** [comment]
Thank you. This error has been occurring intermittently, and I haven't yet been able to discern a pattern in when it shows up. On this occasion, rebooting my computer fixed it but, as I recall, that hasn't always worked in the past. Anyway, the next time it happens, I'll run "netstat -a -b" (which I just learned about) and check if that port is already open.

Thanks for a great package. I've switched over almost completely from Jupyter Notebook to Emacs and org/ob-ipython.
